### ISSUE TITLE
Copy-Paste Markers doesn't place them at the correct Timecode

### ISSUE THREAD

**Issue by samplue** [latenitefilms/MarkerToolbox#28]
If I Command+C markers in the Edit Index and then paste them (Option+V) in the active timeline, the markers will be added at a different Timecode. See the Before/After screenshots. Might be a bug in FCP?! 

Ideally I'd want to place my markers on the primary storyline, copy-pasting would be the easiest and fastest way. 

![Sam_2023-12-16_11h55m10](https://github.com/latenitefilms/MarkerToolbox/assets/78317416/c0c05b49-0ae2-43cf-90f8-e110ad883f36)
![Sam_2023-12-16_11h55m24](https://github.com/latenitefilms/MarkerToolbox/assets/78317416/d068767a-44c5-4933-af30-aee38d24a4dd)

**Comment by latenitefilms** [comment]
No idea to be honest... but I don't think this has anything to do with Marker Toolbox?

**Comment by samplue** [comment]
True, not entirely related to Marker Toolbox and possibly a FCP bug. I just thought that it might be possible to add the Markers directly onto the Primary Storyline instead of importing a compound clip that contains the markers.

**Comment by latenitefilms** [comment]
You can always break apart the Compound Clip?
